src/sed/core/processor.py

```diff
@@ -2375,13 +2377,25 @@ def get_normalization_histogram(
                     axis,
                     self._binned.coords[axis].values,
                     self._config["dataframe"]["timed_dataframe_unit_time"],
+                    hist_mode=self.config["binning"]["hist_mode"],
+                    mode=self.config["binning"]["mode"],
+                    pbar=self.config["binning"]["pbar"],
+                    n_cores=self.config["core"]["num_cores"],
+                    threads_per_worker=self.config["binning"]["threads_per_worker"],
+                    threadpool_api=self.config["binning"]["threadpool_API"],
                 )
             else:
                 self._normalization_histogram = normalization_histogram_from_timed_dataframe(
                     self._timed_dataframe,
                     axis,
                     self._binned.coords[axis].values,
                     self._config["dataframe"]["timed_dataframe_unit_time"],
+                    hist_mode=self.config["binning"]["hist_mode"],
```

Comment:
This seems repeated. Probably can go out of the loop

Reply:
I changed the structure now to repeat less code